docs/rule/no-obsolete-elements.md

```diff
@@ -0,0 +1,66 @@
+## no-obsolete-elements
+
+Some elements are entirely obsolete and must not be used by authors. 
+
+This rule forbids the use of obsolete elements.
+
+### Examples
+
+This rule **forbids** the following:
+
+```hbs
+  <applet></applet>
+  <acronym></acronym>
+  <bgsound></bgsound>
+  <dir></dir>
+  <frame></frame>
+  <frameset></frameset>
+  <noframes></noframes>
+  <isindex></isindex>
+  <keygen></keygen>
+  <listing></listing>
+  <menuitem></menuitem>
+  <nextid></nextid>
+  <noembed></noembed>
+  <plaintext></plaintext>
+  <rb></rb>
+  <rtc></rtc>
+  <strike></strike>
+  <xmp></xmp>
+  <basefont></basefont>
+  <big></big>
+  <blink></blink>
+  <center></center>
+  <font></font>
+  <marquee></marquee>
+  <multicol></multicol>
+  <nobr></nobr>
+  <spacer></spacer>
+  <tt></tt>
+```
+
+This rule **allows** the following:
+
+```hbs
```

Comment:
I'd probably remove this listing, its nearly impossible to be exhaustive and the main point is "it works with everything else"